src/environments/environment.ts

```diff
@@ -3,7 +3,9 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  pusher: {
+    key: '3397df7b343c3c05e778',
```

Comment:
This should also be in your .env file. And you should retrieve it using `dotenv` and process.env